src/views/List.jsx

```diff
@@ -22,7 +26,22 @@ export function List({ data }) {
 		setFilterInput('');
 	};
 
-	return (
+	const handleAddItem = () => {
+		/* navigate to add item view */
+		navigate('/add-item');
+	};
+
+	return data.length == 1 ? (
```

Comment:
At first I was wondering why an empty list is equivalent to `data.length === 1` here. But I realized it was because every time a list is created an empty document is automatically generated. Like Sarah, I wonder how to make it so that no empty document is generated upon creation of the list.

Reply:
I think you should consider "sanitizing" the data state value upstream (i.e. filtering data array to be only non-empty items). This way you could use the data.length === 0 case.

Reply:
@mtfoley Hey Matt. Thanks for the suggestions.  Could you clarify  a bit more what you mean by sanitizing upstream?  We've been experimenting, but no dice yet.  Thanks!

Reply:
Randall and I worked on it for a few hours earlier, but I'm now using map and includes and I'm getting somewhere!

Reply:
This is the route we ended up going with, feel free to let us know if there's any changes we should make!

`	/* This function will return a boolean value of whether the list
	is empty or not */
	const checkForEmptyList = () => {
		//return true if list is empty and false if list contains items
		const shoppingListArr = data.map((item) => item.name);
		return shoppingListArr.includes(undefined) ? true : false;
	};`